src/lib/gadgets/gadgets.ts

```diff
@@ -223,15 +373,15 @@ const Gadgets = {
    * @example
    * ```ts
    * const x = Provable.witness(Field, () => Field(0b001100)); // 12 in binary
-   * const y = Gadgets.rightShift(x, 2); // right shift by 2 bits
+   * const y = Gadgets.rightShift64(x, 2); // right shift by 2 bits
    * y.assertEquals(0b000011); // 3 in binary
    *
    * const xLarge = Provable.witness(Field, () => Field(12345678901234567890123456789012345678n));
    * rightShift(xLarge, 32); // throws an error since input exceeds 64 bits
```

Comment:
two questions:
1. will we have a leftShift64 operation?
2. should this comment be updated to rightShift64 or the second part of the example is no longer needed?
```suggestion
   * rightShift64(xLarge, 32); // throws an error since input exceeds 64 bits
```